db/migrate/20151110191852_create_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRedirections < ActiveRecord::Migration
+  def change
+    create_table :redirections do |t|
+      t.timestamps null: false
+      t.references :next
+      t.string :slug, index: true
+      t.string :url, length: 255
```

Comment:
> we can has some whack-ass long urls.

What if we bypass that entirely and use a `text` instead of a `string`? Then we could put up to 2GB of text in that column.